infra/app/app-config/main.tf

```diff
@@ -3,7 +3,7 @@ locals {
   environments                    = ["dev", "staging", "prod"]
   project_name                    = module.project_config.project_name
   image_repository_name           = "${local.project_name}-${local.app_name}"
-  has_database                    = false
+  has_database                    = true
```

Comment:
Just realized that I'm surprised that this doesn't break the template-only-ci-infra.yml workflow. My understanding is that workflow will go through all the setup steps but it won't actually set up the database layer. So wouldn't it error out when it instantiates the service layer?